source/firstprogram-social-interventions.rst

```diff
@@ -0,0 +1,32 @@
+Introduction of Social Interventions
+====================================
+
+Various social interventions can be made such as quarantine as explained in the previous section, or vaccine drives. 

```

Comment:
I don't like this line, replace it with something like:

"BharatSim allows the user the possibility to implement a variety of social interventions that modify disease spread at the level of individuals, like quarantines, lockdowns, or vaccination drives. In this section, we will describe a simple implementation of a quarantine."